hibernate-core/src/test/java/org/hibernate/test/annotations/any/AnyEagerHbmTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.test.annotations.any;
```

Comment:
Since it's a unit test for HBM, not for annotation, can you move this unit test out of the annotations package?

Reply:
Bu then all common classes will have to be moved as well.

Reply:
Hmm... I guess all of the test cases are kind of messy. So be it.

Reply:
I've moved those to `org.hibernate.test.any`.